pubsub/api/Pubsub.Samples/PullMessagesAsyncWithDeliveryAttempts.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_dead_letter_delivery_attempt]
+
+using Google.Cloud.PubSub.V1;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class PullMessagesAsyncWithDeliveryAttemptsSample
+{
+    public async Task<int> PullMessagesAsyncWithDeliveryAttempts(string projectId, string subscriptionId, bool acknowledge)
+    {
+        // This is an existing subscription with a dead letter policy.
+        SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId, subscriptionId);
+
+        SubscriberClient subscriber = await SubscriberClient.CreateAsync(subscriptionName);
+
+        int messageCount = 0;
+        Task startTask = subscriber.StartAsync((PubsubMessage message, CancellationToken cancel) =>
```

Comment:
```suggestion
        await subscriber.StartAsync((PubsubMessage message, CancellationToken cancel) =>
```
Let's wait for the subscriber to start so we don't loose some of our 5 seconds waiting for it.